src/plugins/transform-classes/index.js

```diff
@@ -580,7 +580,11 @@ export default function({ types: t }) {
 
 		if (t.isVariableDeclarator(inv.parent) && t.isNodesEquivalent(inv.parent.id, id)) {
 			if (inv.parentPath.parent.declarations.length === 1) {
-				inv.parentPath.parentPath.replaceWith(t.classDeclaration(id, superD, body));
+				try {
```

Comment:
Can you update this PR to remove this change? It's from #6 and seems to cause issues as you noted there.

Reply:
@developit done